ci/env/kava-protonet/genesis.json

```diff
@@ -865,6 +865,9 @@
               {
                 "@type": "/kava.committee.v1beta1.TextPermission"
               },
+              {
+                "@type": "/kava.committee.v1beta1.CommunityCDPWithdrawCollateralPermission"
```

Comment:
ha, i added this late to my other pr, standby for a simple merge conflict :trollface: